sky/shell/BUILD.gn

```diff
@@ -3,9 +3,21 @@
 # found in the LICENSE file.
 
 import("//build/config/ui.gni")
+import("//mojo/dart/embedder/embedder.gni")
+
+dart_embedder_resources("generate_sky_embedder_diagnostic_server_resources_cc") {
+  inputs = [
```

Comment:
It looks like the dart_embedder_resources template (imported from Mojo) doesn't support visibility